src/Components/Dashboard/CreateNFT/createNft.js

```diff
@@ -164,6 +164,10 @@ const CreateNft = (props) => {
     let nftDetail = { ...details };
     nftDetail.attributes = formValues;
     nftDetail.owner_id = user.user_id;
+    nftDetail.tracker = {
+      id: '13', // random value. it may need to be static or linked to a specific customer in the future
```

Comment:
Can we auto generate or it should be fixed? If it is fixed value, lets store this in constant file.

Reply:
as mentioned in the comment, the id field is reserved for future use. we might get this value from the advertiser track URL or this might be static to a specific customer. this field is not being used for now as well.

Reply:
Okay